cohorts/functions.py

```diff
@@ -258,6 +303,36 @@ def expressed_neoantigen_count(row, cohort, filter_fn, normalized_per_mb, **kwar
                             only_expressed=True,
                             **kwargs)
 
+@use_defaults
```

Comment:
I thought about this too, but didn't really have time to rework these. For now just need to get the analysis done. Up to you if you want to leave this PR hanging until then.